src/test/ui/issues/issue-74224.rs

```diff
@@ -0,0 +1,11 @@
+struct A<T> {
```

Comment:
There has been an effort to organize tests by the area of compiler they exercise, and we try to avoid additional tests into `issues` directory.

Maybe we could have a new directory for structural recursion check, if no other directory is appropriate?